l3backend/l3backend-color.dtx

```diff
@@ -1308,6 +1308,16 @@ if luaotfload and luaotfload.set_transparent_colorstack then
     local html = htmlcolor:match(value)
     if html then return html end
 
+% If no l3color named color with this name is known, check for defined xcolor colors
+    local l3color_prop = token.get_macro(string.format('l__color_named_%s_prop', value))
```

Comment:
Similar to commit 096d4ec6 (Use @@ convention in Lua opacity backend code, 2023-11-05). Current file has set
https://github.com/latex3/latex3/blob/69a354502548f0a9ff3f4df8b906d971f86b4fab/l3backend/l3backend-color.dtx#L60

```suggestion
    local l3color_prop = token.get_macro(string.format('l_@@_named_%s_prop', value))
```